names/module.py

```diff
@@ -0,0 +1,250 @@
+from typing import Optional
+
+import discord
+from discord.ext import commands
+
+from pie import check, utils, i18n, logger
+from modules.base.admin.database import BaseAdminModule
+from modules.boards.karma.database import KarmaMember
+
+from .database import Price
+
+_ = i18n.Translator("modules/fun").translate
+bot_log = logger.Bot.logger()
+
+
+class Names(commands.Cog):
+    """Pay by karma and I change your name."""
+
+    def __init__(self, bot: commands.Bot):
+        # Check if dependency module is loaded.
+        if not BaseAdminModule.get("boards.karma").enabled:
+            raise Exception(
+                "Unable to load names module due to missing module `boards.karma`."
+            )
+
+        Price.add_commnand("set", 500)
+        Price.add_commnand("unset", 200)
```

Comment:
After the database changes are done, this should not be necessary. There should be no defaults -- if it is not set, the nicknames commands should not work.